app/src/main/java/br/com/nespressohelper/daitan/nespressohelper/MainActivity.java

```diff
@@ -51,185 +72,690 @@
+//        coffee.setImageId(R.drawable.cafematinal);
+
+        drawable = getDrawable(R.drawable.cafematinal);
+        bitmap = ((BitmapDrawable) drawable).getBitmap();
+        stream = new ByteArrayOutputStream();
+        bitmap.compress(Bitmap.CompressFormat.PNG, 100, stream);
+        bitmapData = stream.toByteArray();
+        coffee.setImage(bitmapData);
+
+        coffees.add(coffee);
+
+        coffee = new Coffee();
+
+        //22
+        coffee.setName("Lungo");
+        coffee.setCapsules(1);
+        coffee.setBars1(4);
+        coffee.setBars2(0);
+        coffee.setDescription("Grãos 100% Arábica com torrefação média, aroma frutado e torrado. Intenso sabor aromático, ligeiramente picante, com notas persistentes no paladar mesmo após degustação. Pode ser degustado no café da manhã ou em qualquer hora do dia.");
+//        coffee.setImageId(R.drawable.lungo);
+
+        drawable = getDrawable(R.drawable.lungo);
+        bitmap = ((BitmapDrawable) drawable).getBitmap();
+        stream = new ByteArrayOutputStream();
+        bitmap.compress(Bitmap.CompressFormat.PNG, 100, stream);
+        bitmapData = stream.toByteArray();
+        coffee.setImage(bitmapData);
+
+        coffees.add(coffee);
+
+        coffee = new Coffee();
+
+        //23
+        coffee.setName("Chai Tea Latte");
+        coffee.setCapsules(2);
+        coffee.setBars1(3);
+        coffee.setBars2(3);
+        coffee.setDescription("A exótica combinação do chá preto, leite cremoso e uma seleta combinação de especiarias: canela, gengibre cardamomo e cravo.");
+//        coffee.setImageId(R.drawable.chaitealatte);
+
+        drawable = getDrawable(R.drawable.chaitealatte);
+        bitmap = ((BitmapDrawable) drawable).getBitmap();
+        stream = new ByteArrayOutputStream();
+        bitmap.compress(Bitmap.CompressFormat.PNG, 100, stream);
+        bitmapData = stream.toByteArray();
+        coffee.setImage(bitmapData);
+
+        coffees.add(coffee);
+
+        coffee = new Coffee();
+
+        //24
+        coffee.setName("Marrakesh Style Tea");
+        coffee.setCapsules(1);
+        coffee.setBars1(6);
+        coffee.setBars2(0);
+        coffee.setDescription("Mergulhe na maravilha do chá marroquino. Uma mistura do aromático chá verde, menta intensamente saborosa e refrescante, coberto com a cremosa espuma e adoçado com perfeição.");
+//        coffee.setImageId(R.drawable.marrakesh);
+
+        drawable = getDrawable(R.drawable.marrakesh);
+        bitmap = ((BitmapDrawable) drawable).getBitmap();
+        stream = new ByteArrayOutputStream();
+        bitmap.compress(Bitmap.CompressFormat.PNG, 100, stream);
+        bitmapData = stream.toByteArray();
+        coffee.setImage(bitmapData);
+
+        coffees.add(coffee);
+
+        coffee = new Coffee();
+
+        dbUpdate update = new dbUpdate();
+
+        for(it = 0; it < coffees.size(); it++) {
+            if(update.addCoffee(coffees.get(it))) {
```

Comment:
You should probably move all of this DB initialisation code from the MainActivity to a separate class.